test/runtests.jl

```diff
@@ -22,6 +22,8 @@ end
 
 println("Testing ChainRules.jl")
 @testset "ChainRules" begin
+    include_test("test_helpers.jl")
+    println()
```

Comment:
```suggestion
```

Reply:
This wa intentional. See the other uses of `println()` below

Reply:
yeah do we want to rethink that since we add `println` in `include_test`? It doesn't really matter

Reply:
out of scope for this PR